praw/models/reddit/subreddit.py

```diff
@@ -14,6 +14,7 @@
 from .mixins import MessageableMixin
 from .modmail import ModmailConversation
 from .wikipage import WikiPage
+from .emoji import Emoji
```

Comment:
Can you keep this in sorted order please?

Reply:
Done